jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/logging/PlainConsoleLoggerTest.java

```diff
@@ -49,12 +49,12 @@ public void testLog() {
     testPlainConsoleLogger =
         new PlainConsoleLogger(messageConsumers.build(), singleThreadedExecutor);
 
-    testPlainConsoleLogger.log(Level.LIFECYCLE, "lifecycle");
-    testPlainConsoleLogger.log(Level.PROGRESS, "progress");
+    testPlainConsoleLogger.log(Level.LIFECYCLE, "\u001B[36;1mlifecycle\u001B[0m");
+    testPlainConsoleLogger.log(Level.PROGRESS, "\u001B[33m progress\u001B[0m");
```

Comment:
perhaps we can remove the spaces here and below: 
```suggestion
    testPlainConsoleLogger.log(Level.PROGRESS, "\u001B[33mprogress\u001B[0m");
```